edx_proctoring/tests/test_api.py

```diff
@@ -1654,6 +1654,25 @@ def test_status_summary_bad(self):
 
         self.assertIsNone(summary)
 
+    @patch('edx_proctoring.backends.tests.test_backend.TestBackendProvider')
+    def test_update_exam_attempt_timed(self, mock_backend):
```

Comment:
I'm suspicious that this mock isn't doing what you want.  tests/__init__.py makes one instance of TestBackendProvider, and uses that for all the tests, I think.  "assert_not_called" may be true just because the mock isn't installed properly.